### PR TITLE
adding code

### PR DESCRIPTION
**Pull request by hanu66**
Hi @yaswanth1106 ,

Created terraform code for azure VM creation, Can you please review the code.

Regards,
Hanu

### CODE REVIEW

File: vm.tf.txt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "${var.prefix}-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_network" "main" {
+  name                = "${var.prefix}-network"
+  address_space       = ["10.0.0.0/16"]
```
Comment (yaswanth1106) [inline vm.tf.txt]: Update range to 10.0.0.0/24

File: vm.tf.txt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  resource_group_name = azurerm_resource_group.example.name
+
+  ip_configuration {
+    name                          = "testconfiguration1"
+    subnet_id                     = azurerm_subnet.internal.id
+    private_ip_address_allocation = "Dynamic"
+  }
+}
+
+resource "azurerm_virtual_machine" "main" {
```
Comment (yaswanth1106) [inline vm.tf.txt:36]: change name to dev server

**Review by yaswanth1106 — COMMENTED** [review]
Hi @hanu66 ,
Can you please work on review comments and update PR.

File: vm.tf.txt (hunk 1 above)
Comment (hanu66) [reply vm.tf.txt]: Updated range..

File: vm.tf.txt (hunk 2 above)
Comment (hanu66) [reply vm.tf.txt:36]: update server namme

**Review by yaswanth1106 — APPROVED** [review]
looks good.

**Pull request merged**